Flutter/calendar/how-to/custom-widget-on-flutterflow.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Calendar widget in FlutterFlow | Syncfusion
-description: You can learn here all about how to adding Syncfusion Flutter Calendar (SfCalendar) widget in FlutterFlow.
+title: Calendar Widget in FlutterFlow | Syncfusion
+description: You can learn here all about adding Syncfusion Flutter Calendar ( SfCalendar Widget ) widget in FlutterFlow.
```

Comment:
You can learn here all about adding Syncfusion Flutter Calendar ( SfCalendar ) widget in FlutterFlow.

Reply:
@AswiniDileep Address in new PR